sublime-package.json

```diff
@@ -17,17 +17,29 @@
                   "enum": [
                     "system",
                     "auto",
-                    "github"
+                    "github",
+                    "custom"
                   ],
                   "enumDescriptions": [
                     "Prefers the system binary found in path",
                     "Prefers the system binary but falls back to GitHub without user intervention",
                     "Prefers the latest tested release from GitHub",
+                    "Use the custom command in the initializationOptions below",
                   ],
                 },
+                "system_binary": {
+                  "type": "string",
+                  "default": "clangd",
+                  "markdownDescription": "The binary to use when `binary` is set to `system`."
+                },
                 "initializationOptions": {
                   "additionalProperties": false,
                   "properties": {
+                    "custom_command": {
+                      "type": "array",
+                      "default": [],
+                      "markdownDescription": "A custom command to start clangd. Set `binary` to `custom` to use this command. The configuration arguments are added to this command."
```

Comment:
I don't think it's quite so clear what "configuration arguments" mean. You could specify more precisely:

> The `clang.*` configuration arguments...

or whatever it really means?

Reply:
I updated the description